ui/app/pages/keychains/index.scss

```diff
@@ -25,6 +25,12 @@
   margin: 60px 0 30px 0;
   position: relative;
   max-width: initial;
+
+  &__input-label {
```

Comment:
There was no rule defined for `import-account__input-label` so I think this rule just was forgotten to be added when this component was made (in line with other text label styling)